service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -37,7 +37,7 @@ public CaseDataExtractionService(DateFormatterService dateFormatterService,
 
         // Validation within our frontend ensures that the following data is present
         return Map.of(
-            "courtName", hmctsCourtLookupConfiguration.getCourt(caseData.getCaseLocalAuthority()).getName(),
+            "courtLocation", getCourtLocation(caseData),
```

Comment:
Please update caseDataValidationServiceTest

Reply:
Tests have now been updated to accommodate for the new location variable in Court